cmake/jemalloc.cmake

```diff
@@ -23,6 +23,15 @@ else()
   set(DISABLE_CACHE_OBLIVIOUS "--disable-cache-oblivious")
 endif()
 
+if (NOT DISABLE_JEMALLOC_PROFILING)
+  set(ENABLE_JEMALLOC_PROFILING "--enable-prof")
+  # jemalloc profiling is enabled by default, but we can disable it and
+  # enable it at runtime by setting the environment variable.
+  set(JEMALLOC_CONFIG_MALLOC_CONF "prof:true,prof_active:false")
```

Comment:
this is a CMake variable and will not be passed to the build phase of jemalloc.

Reply:
Seems it can also be configured in runtime via `MALLOC_CONF` env var, so we can ignore it?